code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -120,6 +120,7 @@
 	flash_act(affect_silicon = 1)
 
 /mob/living/silicon/bullet_act(obj/item/projectile/P, def_zone)
```

Comment:
redid how stuff works here because the rng stuff was frankly Fucking Terrible so pointing this out before someone screeches balance change

Reply:
notably: stuff is now knocked down from cyborgs consistently if a projectile has either 10 dmage on it or 20 stamina damage on it